File: src/com/github/luomingxu/idea/ClipboardUtil.java
package com.github.luomingxu.idea;

import java.awt.*;
import java.awt.datatransfer.Clipboard;
import java.awt.datatransfer.StringSelection;
import java.awt.datatransfer.Transferable;

public class ClipboardUtil {

    public static void set(String txt) {
        Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
        Transferable trans = new StringSelection(txt);
        clipboard.setContents(trans, null);
    }
}

File: src/com/github/luomingxu/idea/util/ClipboardUtil.kt
package com.github.luomingxu.idea.util

import java.awt.Toolkit
import java.awt.datatransfer.StringSelection
import java.awt.datatransfer.Transferable

object ClipboardUtil {
    fun set(txt: String?) {
        txt?.let {
            val clipboard = Toolkit.getDefaultToolkit().systemClipboard
            val trans: Transferable = StringSelection(it)
            clipboard.setContents(trans, null)
        }
    }
}
